electron-app/src/store/session/getters.ts

```diff
@@ -15,6 +15,14 @@ export const getters: GetterTree<SessionState, RotkehlchenState> = {
     return state.accountingSettings.lastBalanceSave;
   },
 
+  privacyMode: (state: SessionState) => {
+    return state.privacyMode;
+  },
+
+  scrambleData: (state: SessionState) => {
```

Comment:
I would suggest to remove the getters here and instead where you do `mapGetters(['privacyMode']) to do `mapState(['privacyMode`]) instead. Since we just need to access the state directly it doesn't make that much sense. You can see that the other getters only expose a property of an object. The idea behind it is that I want to avoid mapping the whole settings object just for one property and thus I created the getters. But for properties that are part of the state, you can map them directly.

Reply:
So just remove `scrambleData` and `privacyMode` entirely from `getters.ts` and replace instances of `mapGetters(['scrambleData'])` and `mapGetters(['privacyMode'])` with `mapState(['scrambleData'])` and `mapState(['privacyMode'])` ?

Reply:
Can I also get rid of the stuff in `mutations.ts`?